compiler/src/typed/checkertypes.re

```diff
@@ -69,6 +69,7 @@ let type_constant =
   | Const_wasmf32(_) => instance_def(Builtin_types.type_wasmf32)
   | Const_wasmf64(_) => instance_def(Builtin_types.type_wasmf64)
   | Const_bigint(_) => instance_def(Builtin_types.type_bigint)
+  | Const_rational(_) => instance_def(Builtin_types.type_rational)
```

Comment:
As I was reading this I thought "he didn't add the `Rational` type. How does any of this work?" but it turns out we knew what we were doing all along 😂

Reply:
Haha yeah not much ended up having to be changed in the compiler